src/main/java/uk/gov/register/resources/ItemResource.java

```diff
@@ -18,11 +19,14 @@
 public class ItemResource {
     private final RegisterReadOnly register;
     private final ViewFactory viewFactory;
+    private final HeaderProvider headerProvider;
 
     @Inject
-    public ItemResource(RegisterReadOnly register, ViewFactory viewFactory) {
+    public ItemResource(RegisterReadOnly register, ViewFactory viewFactory,
+                        final RequestContext requestContext) {
```

Comment:
indentation is weird here?

Reply:
Parameters are aligned, is it weird?

Reply:
OK, yes, that is fine, think I was just confused how it was looking on github,